amtl/experimental/am-argparser.cc

```diff
@@ -0,0 +1,38 @@
+// vim: set sts=8 ts=4 sw=4 tw=99 et:
```

Comment:
while likely intentional - I think this is the first .cc file in the tree.

Reply:
Hrm, good catch. I will switch it to .cpp.